lib/internal/main/watch_mode.js

```diff
@@ -64,11 +66,23 @@ let exited;
 
 function start() {
   exited = false;
+
+  let kEnv = {};
+  for (const arg of process.execArgv) {
+    if (StringPrototypeStartsWith(arg, '--env-file')) {
+      const envFile = resolve(StringPrototypeSplit(arg, '=')[1]);
+      watcher.filterFile(envFile, null);
+      const envContent = parseEnv(readFileSync(envFile, 'utf8'));
+      kEnv = { ...kEnv, ...envContent };
```

Comment:
If this implementation is correct, there is no need to make an unnecessary copy. We can just use `Object.assign(kEnv, envContent)`

Reply:
See my comment above.the child process should handle the flag

Reply:
Ah, my bad!